test/nbrowser/CreateTeamSite.ts

```diff
@@ -0,0 +1,62 @@
+import { assert, driver, Key } from 'mocha-webdriver';
+import * as gu from 'test/nbrowser/gristUtils';
+import { cleanupExtraWindows, setupTestSuite } from 'test/nbrowser/testUtils';
+
+describe('Create Team Site', function () {
+  this.timeout(20000);
+  cleanupExtraWindows();
```

Comment:
Does this test leave extra windows open?

Reply:
Yes, when the test is finished the extra windows is closed